demisto_sdk/commands/lint/linter.py

```diff
@@ -985,22 +985,25 @@ def _docker_run_linter(
         exit_code = SUCCESS
         output = ""
         command = [self._facts["lint_to_commands"][linter]]
-        try:
-            container: docker.models.containers.Container = (
-                get_docker().create_container(
-                    name=container_name,
-                    image=test_image,
-                    command=command,
-                    user=f"{os.getuid()}:4000",
-                    files_to_push=[(self._pack_abs_dir, "/devwork")],
-                    environment=self._facts["env_vars"],
+        for trial in range(3):
+            try:
+                container: docker.models.containers.Container = (
+                    get_docker().create_container(
+                        name=container_name,
+                        image=test_image,
+                        command=command,
+                        user=f"{os.getuid()}:4000",
+                        files_to_push=[(self._pack_abs_dir, "/devwork")],
+                        environment=self._facts["env_vars"],
+                    )
                 )
-            )
-        except Exception:
-            logger.exception(
-                f"{log_prompt} - could not create container: {container_name=}, {test_image=}, {command=}"
-            )
-            raise
+                break
+            except Exception:
+                if trial == 2:
```

Comment:
use a constant for the attempt count, then `if trial == ATTEMPT_COUNT - 1`, clearer than hardcoded numbers (also, `attempt` can be a clearer name than `trial` imo)